app/models/host.rb

```diff
@@ -1693,11 +1693,6 @@ def self.host_create_os_types
   end
   Vmdb::Deprecation.deprecate_methods(self, :host_create_os_types)
 
-  def has_compliance_policies?
-    _, plist = MiqPolicy.get_policies_for_target(self, "compliance", "host_compliance_check")
-    !plist.blank?
-  end
```

Comment:
Why was this removed?  I expected it to be similar to the others where the method was changed.

Reply:
Oh nvm - I didn't realize there was a common base method below.